packages/components/src/Form/Fields/FieldToggleSwitch/FieldToggleSwitch.spec.tsx

```diff
@@ -50,7 +50,7 @@ describe('FieldToggleSwitch', () => {
     const input = screen.getByDisplayValue('example')
     const id = input.getAttribute('aria-describedby')
     expect(id).toBeDefined()
-    expect(id).toEqual('describedby-test')
+    expect(id).toEqual('test-description')
```

Comment:
In the interest of reducing downstream snapshot changes I think we should go back to the old format (either's fine, not changing means less "noise" in snapshots and changeset here). Update incoming.

```suggestion
    expect(id).toEqual('describedby-test')
```